<launch>
  <group ns="rarm">
    <include file="$(find multi_device_view)/launch/recognition/apriltag/apriltag_rarm.launch"/>
  </group>

  <group ns="larm">
    <include file="$(find multi_device_view)/launch/recognition/apriltag/apriltag_larm.launch"/>
  </group>

  <group ns="module_0">
    <include file="$(find multi_device_view)/launch/recognition/apriltag/apriltag_module_0.launch"/>
  </group>

  <group ns="module_1">
    <include file="$(find multi_device_view)/launch/recognition/apriltag/apriltag_module_1.launch"/>
  </group>
  
</launch>
